src/jvmTest/kotlin/bdd/SpaceEngineersCucumberTest.kt

```diff
@@ -59,6 +63,11 @@ class SpaceEngineersCucumberTest {
             check(it is WorldController)
             it.load(File("$SCENARIO_DIR$scenarioId").absolutePath)
         }
+        //all block are new for first request
```

Comment:
We should set some convention for comments. At minimum, I strongly prefer space after //.

Other aspects are less clear, but I'm used to using capital letter and the dot at the end. Plus there are some mistakes. So I'd write:

`// All blocks are new for the first request.`

Reply:
Yeah agreed.